### PR TITLE
*: Add readme

### PR DESCRIPTION
**Pull request by shenli**
@siddontang @qiuyesuifeng @ngaut PTAL

### CODE REVIEW

**Comment by ngaut** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# TiDB Documentation
+
+TiDB is a distributed SQL database. Inspired by the design of Google F1, TiDB supports the best features of both traditional RDBMS and NoSQL.
+
+## Getting started
+ - [Quick Start for TiDB](https://github.com/pingcap/tidb/blob/master/docs/QUICKSTART.md)
+
+## Operating TiDB clusters
+
+ - [Supported platform](op-guide/build.md#supported-platforms)
```
Comment (siddontang) [inline README.md]: no user build refer?

Comment (shenli) [inline README.md]: PTAL @siddontang

**Comment by siddontang** [comment]
LGTM

File: README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# TiDB Documentation
```
Comment (queenypingcap) [inline README.md]: +# Documentation for the TiDB project

Comment (shenli) [inline README.md]: If change this to Documentation, we should also change the following short introduction.

**Pull request merged**